bin/http-server

```diff
@@ -32,6 +33,7 @@ if (argv.h || argv.help) {
     '  -U --utc     Use UTC time format in log messages.',
     '',
     '  -P --proxy   Fallback proxy if the request cannot be resolved. e.g.: http://someurl.com',
+    '  --auto       Set proxy autoRewrite flag for proxy, handling 301/302 responses',
```

Comment:
Instead of changing `auto` to `autoRewrite` later, I think it would be simpler and more clear to use the option `--auto-rewrite` from the start instead. Otherwise I think this looks good.